man/chpl.txt

```diff
@@ -195,6 +195,12 @@ OPTIONS
   --[no-]bounds-checks   Enable [disable] run-time bounds checking,
                     e.g. during slicing and array indexing.
 
+  --[no-]formal-domain-checks   Enable [disable] run-time checks to ensure
```

Comment:
Oh, good catch.  I thought that the --no-checks flag said "flips all other flags in this man page section" -- and think it should.  Maybe I'll change it.  It appears (based on the compiler source) that it's currently true.  I have a nagging memory that maybe Kyle or Ben recently added a check that --no-checks didn't/shouldn't turn off, but maybe it didn't have a flag like this?

Yeah, good point about the test that will need updating.  That would get caught by the run I'm currently doing.